packages/state-transition/src/block/processAttestationsAltair.ts

```diff
@@ -168,9 +201,33 @@ export function getAttestationParticipationStatus(
   const isMatchingTarget = byteArrayEquals(data.target.root, rootCache.getBlockRoot(data.target.epoch));
 
   // a timely head is only be set if the target is _also_ matching
-  const isMatchingHead =
+  // In gloas, this is called `is_matching_blockroot`
```

Comment:
`is_matching_blockroot` doesn't exist in the spec

Reply:
Looks like it is renamed to `head_root_matches` in 1.6.1